### ISSUE TITLE
Unable to activate plugin

### ISSUE THREAD

**Issue by kevinblake** [the-markup/smol-links#34]
Activation on WP 6.5.3 fails with the following error

```
Warning: require_once(<path_to_wordpress_dir>/wp-content/plugins/smol-links/vendor/composer/autoload_real.php): Failed to open stream: No such file or directory in <path_to_wordpress_dir>/wp-content/plugins/smol-links/vendor/autoload.php on line 23

Fatal error: Uncaught Error: Failed opening required '<path_to_wordpress_dir>/wp-content/plugins/smol-links/vendor/composer/autoload_real.php' (include_path='.:') in <path_to_wordpress_dir>/wp-content/plugins/smol-links/vendor/autoload.php:23 Stack trace: #0 <path_to_wordpress_dir>/wp-content/plugins/smol-links/smol-links.php(25): require_once() #1 <path_to_wordpress_dir>/wp-admin/includes/plugin.php(2389): include_once('/home/dh_wqc2ng...') #2 <path_to_wordpress_dir>/wp-admin/includes/plugin.php(675): plugin_sandbox_scrape('smol-links/smol...') #3 <path_to_wordpress_dir>/wp-admin/plugins.php(60): activate_plugin('smol-links/smol...', 'https://friends...', false) #4 {main} thrown in <path_to_wordpress_dir>/wp-content/plugins/smol-links/vendor/autoload.php on line 23
```

**Comment by dphiffer** [comment]
Thanks for this bug report. I think I see what's going on...

**Comment by kevinblake** [comment]
Thank you, let me know if I can help with more details or any re-testing!

**Comment by dphiffer** [comment]
I think this should fix it: https://github.com/the-markup/smol-links/pull/36

I'll let you know once that gets released!

**Comment by BatMiles** [comment]
@kevinblake, that's been released in the 0.3.1 patch! Give it another go and let us know if it's still broken.

**Comment by kevinblake** [comment]
Great - thank you both for getting to it so quickly - that's working perfectly now!
